.github/CODEOWNERS

```diff
@@ -121,5 +121,5 @@
 /Gems/Twitch/ @o3de/sig-network-maintainers
 
 # SIG-Graphics-Audio ownership areas
-/Gems/Atom*/ @o3de/sig-graphics-audio-maintainers @o3de/sig-graphics-audio-reviewers
-/Gems/Terrain/ @o3de/sig-graphics-audio-maintainers @o3de/sig-graphics-audio-reviewers
+/Gems/Atom*/ @o3de/sig-graphics-audio-maintainers @o3de/sig-graphics-audio-reviewers @o3de/sig-graphics-audio
+/Gems/Terrain/ @o3de/sig-graphics-audio-maintainers @o3de/sig-graphics-audio-reviewers @o3de/sig-graphics-audio
```

Comment:
You can remove `@o3de/sig-graphics-audio`, you only need the `reviewers` and `maintainers` groups.